packages/@aws-cdk/cfnspec/spec-source/specification/000_cfn/000_official/000_AWS_SNS.json

```diff
@@ -41,6 +41,12 @@
           "Required": false,
           "UpdateType": "Mutable"
         },
+        "FilterPolicyScope": {
```

Comment:
Just for testing. Will wait until cloudspec is up to date before ready for review.